deepset_cloud_sdk/_api/upload_sessions.py

```diff
@@ -178,6 +179,12 @@ async def close(self, workspace_name: str, session_id: UUID) -> None:
                 f"Failed to close the upload session. Status code: {response.status_code}."
             )
 
+    @retry(
```

Comment:
thought: We could add retry logging for this method and the list method, [example](https://stackoverflow.com/questions/64002627/python-tenacity-log-exception-on-retry). Will defer to you @ArzelaAscoIi  on this one.

Reply:
we could add this, but it might also just be internal information we wouldn't need to report 🤔  not sure